components/create-collective/sections/CreateCollectiveForm.js

```diff
@@ -65,20 +73,22 @@ class CreateCollectiveForm extends React.Component {
   }
 
   handleChange(fieldname, value) {
-    const collective = {};
-
-    collective[fieldname] = value;
-
-    this.setState({ collective });
+    this.setState(state => ({
+      ...state,
```

Comment:
If I remember `...state` is not necessary. It is when using hooks but not class components.